packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -223,7 +231,9 @@ class _GoogleMapState extends State<GoogleMap> {
       'polygonsToAdd': serializePolygonSet(widget.polygons),
       'polylinesToAdd': serializePolylineSet(widget.polylines),
       'circlesToAdd': serializeCircleSet(widget.circles),
+      '_web_only_mapCreationId': _web_only_mapCreationId,
```

Comment:
Can we test if this ID is created properly?

Reply:
I'll look into adding a test for this, I can make the creation ID visibleForTesting and instantiate this twice. AFAIK there are no unit tests for this file at all in the package.

Reply:
Added a new test for this. (I think this is the first test that mocks the platform interface, instead of the MethodChannels in this package)